ui/src/client/Clients.tsx

```diff
@@ -119,6 +130,11 @@ const Row: SFC<IRowProps> = ({name, value, fEdit, fDelete}) => (
                 style={{display: 'flex', alignItems: 'center', width: 200}}
             />
         </TableCell>
+        <TableCell align="right" padding="none">
+            <IconButton onClick={() => {copyToClipboard(value)}} className="copy" title="Copy to clipboard">
```

Comment:
The end-to-end test currently fail, probably because the column index is out of sync here
https://github.com/gotify/server/blob/9d4e37aa87e5aea5d04b9a6285ca807099c46fbd/ui/src/tests/client.test.ts#L17-L22